db/config.c

```diff
@@ -394,9 +394,7 @@ static char *legacy_options[] = {
     "dont_superset_foreign_keys",
     "enable_sql_stmt_caching none",
     "enable_tagged_api",
-    "env_messages",
     "init_with_time_based_genids",
-    "legacy_schema on",
```

Comment:
Should we keep it on in 7.0, but repurpose the tunable for 7.1 features (e.g., uuid, autoincrement, etc.) in master/7.1 branch?

Reply:
Good idea. Another option would be to have individual tunables to control new features.

Reply:
I prefer new tunables for new features. Old tunables should control exactly what old branch did. Do not want to reuse these parameters.

Reply:
I thought you preferred no new tunables.